kuma/users/admin.py

```diff
@@ -30,39 +18,25 @@ class UserBanAdmin(admin.ModelAdmin):
 admin.site.register(UserBan, UserBanAdmin)
 
 
-class ProfileAdmin(admin.ModelAdmin):
-
-    list_display = ('user_name', 'related_user', 'fullname', 'title',
-                    'organization', 'location', 'content_flagging_email',
-                    'tags')
-
-    list_editable = ('content_flagging_email', 'tags', )
-
-    search_fields = ('user__username', 'homepage', 'title', 'fullname',
+class UserAdmin(BaseUserAdmin):
+    """
+    Extends the admin view of users to show date_joined field
+    add a filter on the field too
+    """
+    list_display = ('username', 'fullname', 'email', 'title', 'organization',
+                    'location', 'content_flagging_email', 'tags',
+                    'date_joined', 'is_staff', 'is_active')
+    list_editable = ('content_flagging_email', 'tags')
+    list_filter = ('is_staff', 'is_superuser', 'is_active', 'date_joined')
+    ordering = ('-date_joined',)
+    search_fields = ('username', 'homepage', 'title', 'fullname',
                      'organization', 'location', 'bio', 'misc',
```

Comment:
blocking: need to remove `'misc'` from this `search_fields` list - it's not part of `User`. Otherwise we get `FieldError: Cannot resolve keyword 'misc' into field.` when searching users in the admin interface.